src/plots/cartesian/axes.js

```diff
@@ -1684,7 +1684,8 @@ axes.doTicks = function(gd, axid, skipTitle) {
                     fontSize*(offsetBase + (ax.showticklabels ? 1.5 : 0.5)));
 
             if(ax.rangeslider && ax.rangeslider.visible && ax._boundingBox) {
-                y += gs.h * ax.rangeslider.thickness + ax._boundingBox.height;
+                y += (fullLayout.height - fullLayout.margin.b - fullLayout.margin.t) *
+                    ax.rangeslider.thickness + ax._boundingBox.height;
```

Comment:
Actually, I think the ideal position for this title is probably somewhere a bit higher than where it ended up - as it is, it looks like it leaves room for tick labels even when there are none. But again, that's not something to address in this PR.